Jenkinsfile

```diff
@@ -41,15 +41,16 @@ node(workerLabel) {
           } // end stage
         } // end docker
       } // end if
-      if (scmVars.GIT_LOCAL_BRANCH ==~ /master|develop/) {
+      checkTag = sh(script: "git describe --tags --exact-match ${scmVars.GIT_COMMIT}", returnStatus: true)
+      if (scmVars.GIT_LOCAL_BRANCH ==~ /master|develop/ || checkTag == 0 ) {
+        docker_push_tags = []
         if (scmVars.GIT_LOCAL_BRANCH == 'master')
-          docker_push_tags = ['latest']
-        else
-          docker_push_tags =  ['develop']
-        checkTag = sh(script: "git describe --tags --exact-match ${scmVars.GIT_COMMIT}", returnStatus: true)
-        if (checkTag == 0)
+          docker_push_tags += ['latest']
+        else if (scmVars.GIT_LOCAL_BRANCH == 'develop')
+          docker_push_tags +=  ['develop']
+        else if (checkTag == 0)
           docker_push_tags += [sh(script: "git describe --tags --exact-match ${scmVars.GIT_COMMIT}", returnStdout: true).trim()]
```

Comment:
`latest` Docker image tag should also be pushed for each tagged release (since tags are always released in a `master` branch)

Reply:
They can decide to create tag (not release) for some feature and it will be pushed to latest docker tag, I think this is not desired behavior.
Currently last commit in `master` always will map to `latest` docker tag.

Reply:
Is `scmVars.GIT_LOCAL_BRANCH` set to `master` when there is a tag created in `master` branch?